lib/htsget-server/docker-compose.yml

```diff
@@ -38,7 +33,13 @@ services:
         target: secret
       - source: selfsigned-site-pem
         target: cert.pem
+      - source: vault-s3-token
+        target: vault-s3-token
     logging: *default-logging
     environment:
       HTSGET_TEST_KEY: "hoodlebug"
+      OPA_SECRET: ${CANDIG_OPA_SECRET}
```

Comment:
Secrets should not be in environment variables

Reply:
moved it to be a Docker secret (but I didn't test it; please make sure this works still)

Reply:
I was able to test this after all...it works fine